### ISSUE TITLE
why the bias is zero

### ISSUE THREAD

**Issue by zealerww** [ZZUTK/Tensorflow-VGG-face#2]
thank you for your work. but i want to know why all the bias of layers is zero. is there somthing wrong?

**Comment by YuDeng** [comment]
If you download the vgg_face.mat from the official website of vgg group, you could find that the bias is zero too. I guess it is because while training, they use batch normalization layers so that there's no need for extra bias.
